src/Mod/Draft/WorkingPlane.py

```diff
@@ -892,17 +892,10 @@ def getPlacement(self, rotated=False):
             and a `Rotation` (`Base::Rotation`).
         """
         if rotated:
-            m = FreeCAD.Matrix(
-                self.u.x, self.axis.x, -self.v.x, self.position.x,
-                self.u.y, self.axis.y, -self.v.y, self.position.y,
-                self.u.z, self.axis.z, -self.v.z, self.position.z,
-                0.0, 0.0, 0.0, 1.0)
+            m = DraftVecUtils.getPlaneRotation(self.u, self.v)
         else:
-            m = FreeCAD.Matrix(
-                self.u.x, self.v.x, self.axis.x, self.position.x,
-                self.u.y, self.v.y, self.axis.y, self.position.y,
-                self.u.z, self.v.z, self.axis.z, self.position.z,
-                0.0, 0.0, 0.0, 1.0)
+            m = DraftVecUtils.getPlaneRotation(self.u, self.v)
```

Comment:
This function is a bit strange now. The `rotated' argument is effectively ignored: `if` and `else` return the same result.

Reply:
Yes, it should have been
```
if rotated:
    m = DraftVecUtils.getPlaneRotation(self.u, self.axis)
else rotated:
    m = DraftVecUtils.getPlaneRotation(self.u, self.v)
```

Reply:
fixed in PR https://github.com/FreeCAD/FreeCAD/pull/9160